src/LinearMap.jl

```diff
@@ -349,7 +350,11 @@ function vertices_list(lm::LinearMap{N})::Vector{Vector{N}} where {N<:Real}
         push!(vlist, lm.M * v)
     end
 
-    return vlist
+    if prune
+        return convex_hull(vlist)
+    else
+        return vlist
+    end
```

Comment:
In cases such as these, the [ternary operator](https://docs.julialang.org/en/v1/manual/control-flow/index.html#man-conditional-evaluation-1) simplifies a bit and i would recommend it: `return prune ? convex_hull(vlist) : vlist`